spec/amp-versioning-policy.md

```diff
@@ -64,6 +64,8 @@ The process for deprecations is similar to the [process for significant changes]
 - Give developers at least 6 weeks after your deprecation warning is in production to apply changes before removing the feature.
 - If you did not include removal in your I2D, file a separate Intent-to-Remove (I2R) when you are ready to remove the deprecated feature.
 
+The given component or integration may be neutered immediately in such a way that usage in a document no longer has side effects, but validation does not break.
```

Comment:
This is only the case if the underlying backend service is being shut down, right?

Reply:
yep. it's being broken at server side already, removing the integration in AMP would not break it further.

Or, it could be that the extension no longer does anything to the page.

Reply:
I think there should be a half-sentence explaining when this may happen (when the underlying backend is being shut down).

Reply:
done